package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@contentful/marketplace-partner-apps",
   "private": false,
   "engines": {
-    "node": ">=16.0.0 < 18.0.0",
-    "npm": ">= 8.0.0 < 9.0.0"
+    "node": ">=18.0.0 < 20.0.0",
+    "npm": ">= 9.0.0 < 10.0.0"
```

Comment:
It might be worth bumping the `npm` range to less than 11? To avoid this potential error https://app.circleci.com/pipelines/github/contentful/apps/26048/workflows/bd464c9e-70e0-4b13-90aa-f9eeafc71878/jobs/80152 with the `circleci/node@5.1.0` I think its using npm -v `10.2.3`